### PR TITLE
Some more exploits that i found

### PR DESCRIPTION
**Pull request by tungdo0602**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -261,14 +261,39 @@ window.webpackChunkdiscord_app.push([[Math.random()], {}, (req) => {for (const m
...
+
+<details>
+  <summary>Give you all nitro user's perks but most of feature are not working (the one that only work is change stream quality)
+</summary>
+
+
+```js
+window.webpackChunkdiscord_app.push([[Math.random()], {}, (req) => {for (const m of Object.keys(req.c).map((x) => req.c[x].exports).filter((x) => x)) {if (m.default && m.default.getCurrentUser !== undefined) {return m.default.getCurrentUser().premiumType = 2;}}}]);
+```
+
```
Comment (hxr404) [inline README.md:289]:
```suggestion
```
This doesn't really work, I don't want people complaining about a broken Nitro "hack".
A working Nitro bypass will be included in Discord Oxygen, so this is redundant anyways

File: README.md [hunk 2]
```diff
@@ -285,7 +310,6 @@ __SECRET_EMOTION__.injectGlobal(`
 
 </details>
 
-</details>
```
Comment (hxr404) [inline README.md:288]: This </details> tag is required, it closes the "Console Hacks" dropdown.
So please re-add it.

File: README.md [hunk 3]
```diff
@@ -102,8 +102,8 @@ The token should be in your Clipboard.
...
+```js&nbsp;
```
Comment (hxr404) [inline README.md:105]:
```suggestion
```js
```
This is redundant

File: README.md [hunk 4]
```diff
@@ -102,8 +102,8 @@ The token should be in your Clipboard.
...
...h([[Math.random()], {}, (req) => {for (const m of Object.keys(req.c).map((x) => req.c[x].exports).filter((x) => x)) {if (m.default && m.default.setToken !== undefined) {return m.default.setToken(e)}if (m.setToken !== undefined) {return m.setToken(e)}}}]);console.log("%cWorked!", "font-size: 50px");}, 50), setTimeout(() => {window.location.reload()}, 2500)}function buttonlogin(){login(document.getElementsByClassName("inputDefault-_djjkz input-cIJ7To")[0].value)}var element;(element=document.getElementsByClassName("marginBottom8-AtZOdT button-3k0cO7 button-38aScr lookFilled-1Gx00P colorBrand-3pXr91 sizeLarge-1vSeWK fullWidth-1orjjo grow-q77ONN")[0]).addEventListener("click",buttonlogin),(element=document.getElementsByClassName("marginBottom20-32qID7")[0]).parentElement.removeChild(element),(element=document.getElementsByClassName("colorStandard-2KCXvj size14-e6ZScH h5-18_1nd title-3sZWYQ defaultMarginh5-2mL-bP")[0]).innerHTML="Token",element.id="Token",(element=document.getElementsByClassName("transitionGroup-aR7y1d qrLogin-1AOZMt")[0]).parentElement.removeChild(element),(element=document.getElementsByClassName("verticalSeparator-3huAjp")[0]).parentElement.removeChild(element);&nbsp;
```
Comment (hxr404) [inline README.md:106]:
```suggestion
function login(e) {setInterval(() => {window.webpackChunkdiscord_app.push([[Math.random()], {}, (req) => {for (const m of Object.keys(req.c).map((x) => req.c[x].exports).filter((x) => x)) {if (m.default && m.default.setToken !== undefined) {return m.default.setToken(e)}if (m.setToken !== undefined) {return m.setToken(e)}}}]);console.log("%cWorked!", "font-size: 50px");}, 50), setTimeout(() => {window.location.reload()}, 2500)}function buttonlogin(){login(document.getElementsByClassName("inputDefault-_djjkz input-cIJ7To")[0].value)}var element;(element=document.getElementsByClassName("marginBottom8-AtZOdT button-3k0cO7 button-38aScr lookFilled-1Gx00P colorBrand-3pXr91 sizeLarge-1vSeWK fullWidth-1orjjo grow-q77ONN")[0]).addEventListener("click",buttonlogin),(element=document.getElementsByClassName("marginBottom20-32qID7")[0]).parentElement.removeChild(element),(element=document.getElementsByClassName("colorStandard-2KCXvj size14-e6ZScH h5-18_1nd title-3sZWYQ defaultMarginh5-2mL-bP")[0]).innerHTML="Token",element.id="Token",(element=document.getElementsByClassName("transitionGroup-aR7y1d qrLogin-1AOZMt")[0]).parentElement.removeChild(element),(element=document.getElementsByClassName("verticalSeparator-3huAjp")[0]).parentElement.removeChild(element);
```

This `&nbsp;` is also redundant

**Review by hxr404 — CHANGES_REQUESTED** [review]
Please address these problems before I merge your PR.
With that many commits, maybe make a new one instead

File: README.md (hunk 2 above)
Comment (tungdo0602) [reply README.md:288]: ok

**Pull request closed without merging**